benchmarking/blackbox_repository/simulated_tabular_backend.py

```diff
@@ -156,9 +156,46 @@ def _run_job_and_collect_results(
                 result[self.elapsed_time_attr] -= elapsed_time_offset
         else:
             results = all_results
+
+        # Makes sure that time is monotonically increasing which may not be the case due to numerical errors or due to
+        # the use of a surrogate
+        for i in range(1, len(results)):
```

Comment:
Did you encounter this? I'd be more happy if we failed here, ideally returning something that would allow us to figure out what is going on.

Reply:
Yes in some cases, the runtime recorded is 0 which causes this error. If we would fail, we should probably fail rather at the creation of the tabular blackbox. I propose to postpone this to another PR given that it seems optional and it would be hard for me to merge otherwise.